src/theme/styles.js

```diff
@@ -0,0 +1,134 @@
+import red from '@material-ui/core/colors/red'
+
+import GastromondWoff2 from './fonts/gastromond-regular.woff2'
+import ProximaNovaWoff2 from './fonts/proxima-nova.woff2'
+import { createMuiTheme } from '@material-ui/core'
+
+const gastromond = {
+  fontFamily: 'Gastromond',
+  fontStyle: 'normal',
+  fontDisplay: 'swap',
+  fontWeight: '400',
+  src: `
+  local('Gastromond'),
+  local('Gastromond-Regular'),
+  url(${GastromondWoff2}) format('woff2')`,
+  unicodeRange:
+    'U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF',
+}
+
+const proxima = {
+  fontFamily: 'Proxima-Nova',
+  fontStyle: 'normal',
+  fontDisplay: 'swap',
+  fontWeight: '400',
+  src: `
+  local('Proxima'),
+  local('Proxima-Nova'),
+  url(${ProximaNovaWoff2}) format('woff2')`,
+  unicodeRange:
+    'U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF',
+}
+
+export const theme = createMuiTheme({
+  typography: {
+    fontFamily: 'Proxima-Nova, sans-serif'
+  },
+  overrides: {
+    MuiCssBaseline: {
+      '@global': {
+        '@font-face': [proxima]
+      }
+    }
+  }
+})
+
+export const styles = (theme) => ({
```

Comment:
I have a hard time accepting JSS. However, I'll admit most of my arguments against it are pretty flimsy. BUT, what about CSS variables?

Reply:
Material UI has a specific way of theming that I will admit is a bit confusing at first. CSS-like variables can be defined with the `createMuiTheme` capability which also allows one to modify the default `theme` values. These variables can then be used anywhere in the project. For example, `theme.layout.spacing(2)`.